lib/core/src/lib/form/components/widgets/display-text/display-text.widget.html

```diff
@@ -1,2 +1,6 @@
-<div [matTooltip]="field.tooltip" matTooltipPosition="above" matTooltipShowDelay="1000"
-    class="adf-display-text-widget {{field.className}}">{{field.value | translate}}</div>
+<div [matTooltip]="field.tooltip"
+     matTooltipPosition="above"
+     matTooltipShowDelay="1000"
```

Comment:
Should possibly be `[matTooltipShowDelay]="1000"`

Reply:
👍